File: Go/K08_1sub.go
package main

func getMaxValue(v1 int, v2 int, v3 int) int {
	var temp int
	if v1 >= v2 {
		temp = v1
	} else {
		temp = v2
	}

	if temp < v3 {
		temp = v3
	}
	return temp
}

func getMinValue(v1 int, v2 int, v3 int) int {
	var temp int
	if v1 < v2 {
		temp = v1
	} else {
		temp = v2
	}

	if temp >= v3 {
		temp = v3
	}

	return temp
}

File: Kotlin/src/K08_1sub.kt
fun GetMaxValue(v1: Int, v2: Int, v3: Int): Int {
    var temp = if (v1 >= v2) v1 else v2
    temp = if (temp >= v3) temp else v3
    return temp
}

fun GetMinValue(v1: Int, v2: Int, v3: Int): Int {
    var temp = if (v1 < v2) v1 else v2
    temp = if (temp < v3) temp else v3
    return temp
}